form.js

```diff
@@ -0,0 +1,18 @@
+function validateEmail(form) {
+  // eslint-disable-next-line no-underscore-dangle
```

Comment:
- [ ] **Optional:** Ensure you do not disable `ESlint` in your chosen solution. This ensures there is conformity in the codebase in your team here at Microverse and in future jobs.